search-sentiments-map/public/scripts/map-script.js

```diff
@@ -164,22 +168,45 @@ function loadCountryData() {
     row.setProperty('country_data', dataVariable);
   });
 
-  const extremaHeader = document.getElementById('extrema-sentiment');
-  extremaHeader.innerText = 
-      'Most Positive Country: ' + countryMax + ', Most Negative Country: ' + countryMin;
-
+  document.getElementById('extrema-sentiment').innerText = 
+      isSentimentMode ?
+          'Most Positive Country: ' + countryMax + 
+              ', Most Negative Country: ' + countryMin :
+          'Most Popular Country: ' + countryMax;
 }
 
-/** 
- * Loads the search interest data for US states on the map. 
- * TODO(chenyuz): Fetch data from the backend to replace the placeholder.
- */
+/** Loads the search interest data for US states on the map. */
 function loadStateData() {
+  // Minimum and maximum scores for current topic. 
+  let dataVariableMax = Number.MIN_VALUE;
+  let dataVariableMin = Number.MAX_VALUE; 
+  // States with the minimum and maximum scores for current topic. 
+  let stateMax = '';
+  let stateMin = '';
+
+  let stateInterests = getCurrentSearchData();
   map.data.forEach(function(row) {
-    let dataVariable = -10;
+    let stateName = row.getProperty('NAME');
+    let interest = stateInterests.filter(stateInterest => 
+        stateInterest.geoName === stateName);
+    let dataVariable = interest.length === 0 ? 
+        NO_RESULTS_DEFAULT_SCORE : interest[0].value[0];
+
+    if (dataVariable > dataVariableMax) {
+      dataVariableMax = dataVariable;
+      stateMax = stateName;
+    }
+    if (dataVariable < dataVariableMin && 
```

Comment:
&& on next line.

Reply:
I just checked the documentation again https://google.github.io/styleguide/jsguide.html#formatting-where-to-break

It says "When a line is broken at an operator the break comes after the symbol. (Note that this is not the same practice used in Google style for Java.)"